site/specs/numbers.json

```diff
@@ -448,8 +445,7 @@
+                    "value": "<!--Messaging-V2 application-->\n<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<ApplicationProvisioningResponse>\n  <Application>\n   <ApplicationId>d4d1b41d-4c05-47d0-838a-34e4f14e4e3e</ApplicationId>\n    <ServiceType>Messaging-V2</ServiceType>\n    <AppName>EBVICs</AppName>\n    <!--DEPRECATED-->\n    <MsgCallbackUrl>https://example.com</MsgCallbackUrl>\n    <InboundCallbackUrl>https://example.com</InboundCallbackUrl>\n    <OutboundCallbackUrl>https://example2.com</OutboundCallbackUrl>\n    <!--DEPRECATED-->\n    <MsgCallbackCreds>\n      <!--DEPRECATED-->\n      <UserId>15jPWZmXdm</UserId>\n      <!--DEPRECATED-->\n      </MsgCallbackCreds>\n    <InboundCallbackCreds>\n      <UserId>15jPWZmXdm</UserId>\n    </InboundCallbackCreds>\n    <OutboundCallbackCreds>\n      <UserId>15jPWZmXdm</UserId>\n    </OutboundCallbackCreds>\n    <RequestedCallbackTypes>\n      <CallbackType>message-delivered</CallbackType>\n      <CallbackType>message-failed</CallbackType>\n      <CallbackType>message-sending</CallbackType>\n    </RequestedCallbackTypes>\n  </Application>\n</ApplicationProvisioningResponse>\n<!--Voice-V2 application-->\n<ApplicationProvisioningResponse>\n<Application>\n  <ApplicationId>d3e418e9-1833-49c1-b6c7-ca1700f79586</ApplicationId>\n    <ServiceType>Voice-V2</ServiceType>\n    <AppName>v1</AppName>\n    <CallInitiatedCallbackUrl>https://a.com</CallInitiatedCallbackUrl>\n    <CallInitiatedMethod>GET</CallInitiatedMethod>\n    <CallStatusCallbackUrl>https://b.com</CallStatusCallbackUrl>\n    <CallStatusMethod>GET</CallStatusMethod>\n    <CallbackCreds>\n      <UserId>login123</UserId>\n    </CallbackCreds>\n    <CallInitiatedFallbackUrl>https://fallback.com</CallInitiatedFallbackUrl>\n    <CallInitiatedFallbackMethod>POST</CallInitiatedFallbackMethod>\n    <CallInitiatedFallbackCreds>\n      <UserId>login2</UserId>\n    </CallInitiatedFallbackCreds>\n    <CallbackTimeout>20</CallbackTimeout>\n  </Application>\n  </ApplicationProvisioningResponse>"
```

Comment:
It works be good to split this into two examples, one for messaging and one for voice. This xml isn't valid